app/controllers/katello/api/registry/registry_proxies_controller.rb

```diff
@@ -77,11 +77,12 @@ def set_user_by_token(token, redirect_on_failure = true)
           redirect_authorization_headers if redirect_on_failure
           return false
         end
+      elsif request.headers['HostUUID']
+        @host = Katello::Host::ContentFacet.find_by(uuid: request.headers['HostUUID'])&.host
```

Comment:
Subscription facet UUID (not content facet) IMO should be the source of truth for host UUID. (They should theoretically be the same, but considering I had to go down a rabbit hole to even figure out where content facet UUID is set..)

Reply:
Are they not set together? I remember hearing somewhere subscription facets were going to go away with SCA eventually or I might be imagining things..

Reply:
No, I think we'd still need some data stored in the subscription facet. No plans to remove the facet.

Yes, they are set together and it probably would work fine. Just would make me feel better to be consistent, since we already use the subscription facet UUID's presence as our defacto "am I a registered host?" litmus test in Katello.

Reply:
Ack..sold..Will update to subscription facet..

Reply:
Ok, I guess we can use content facet.

Can you please add a comment somewhere that content facet UUID should be the same as subscription facet UUID?